### PR TITLE
update collectrnaseqmetrics memory requirement

### PR DESCRIPTION
**Pull request by rannick**
Update memory requirements

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnafusion/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, also make a PR on the nf-core/rnafusion _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by rannick** [comment]
To be merged once linting works (template 2.13.1 PR merged with dev)

File: modules/local/picard/collectrnaseqmetrics/main.nf [hunk 1]
```diff
@@ -33,15 +33,15 @@ process PICARD_COLLECTRNASEQMETRICS {
...
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def avail_mem = 3
+    def avail_mem = 3072
     if (!task.memory) {
         log.info '[Picard CollectRnaMetrics] Available memory not known - defaulting to 3GB. Specify process memory requirements to change this.'
     } else {
-        avail_mem = task.memory.giga
-    }
+        avail_mem = (task.memory.mega*0.8).intValue()
+        }
```
Comment (jemten) [inline modules/local/picard/collectrnaseqmetrics/main.nf:41]: Small thing but could you align the final `}`

**Review by jemten — APPROVED** [review]
Would be good to consider changing to the nf-core version for this module

**Review by apeltzer — APPROVED** [review]

**Comment by rannick** [comment]
Replacement by the nf-core module is planned.

**Pull request merged**